genai-perf/genai_perf/inputs/converters/base_converter.py

```diff
@@ -78,7 +76,7 @@ def _select_model_name(self, index: int) -> str:
                 f"Model selection strategy '{self.config.endpoint.model_selection_strategy}' is unsupported"
             )
 
-    def _get_max_tokens(self, optional_data: Dict[Any, Any]) -> int:
+    def _get_max_tokens(self, optional_data: Dict[Any, Any]) -> Union[int, None]:
```

Comment:
This is a signature change. Why was None added?

Reply:
Previously, we had a default output token mean that was [an integer](https://github.com/triton-inference-server/perf_analyzer/pull/357/files/1c2a07257ede5c4b186822c003935b41160a12b8) (-1). Now the default is None. That default was added as part of the Config changes.

So if we're returning the default value, it is equal to None.